docs/manual/docs/examples/spo/remove-site-scripts.md

```diff
@@ -0,0 +1,64 @@
+# Delete custom SharePoint Site Scripts
+
+Author: [Laura Kokkarinen](https://laurakokkarinen.com/does-it-spark-joy-powershell-scripts-for-keeping-your-development-environment-tidy-and-spotless/#delete-all-sharepoint-site-designs-and-site-scripts)
+
+Site designs and especially site scripts can be something that ends up just hanging around in your tenant for a long time even though you no longer need them for anything. Use the scripts below to get rid of them. You might also find some site scripts that are not linked to any site design and hence never get executed!
+
+```powershell tab="PowerShell Core"
+$sparksjoy = "Project Site", "Issues List"
+$siteScripts = o365 spo sitescript list -o json |ConvertFrom-Json
+$siteScripts = $siteScripts | where {  -not ($sparksjoy -contains $_.Title)}
+if ($siteScripts.Count -eq 0) { break }
+$siteScripts | Format-Table Title, SiteScriptIds, Description
+Read-Host -Prompt "Press Enter to start deleting (CTRL + C to exit)"
+$progress = 0
+$total = $siteScripts.Count
+foreach ($siteScript in $siteScripts) 
+{
+    $progress++
+    write-host $progress / $total":" $siteScript.Title
+    o365 spo sitescript remove -i $siteScript.Id --confirm    
+}
+```
+
+```bash tab="Bash"
+
+#!/bin/bash
+
+# requires jq: https://stedolan.github.io/jq/
+
+sparksjoy=("Project Site"  "Issues List")
+sitesscriptstoremove=()
+while read script; do
+ scriptTitle=$(echo ${script} | jq -r '.Title')
+  exists=true
+  for keep in "${sparksjoy[@]}"; do
+    if [ "$keep" == "$scriptTitle" ] ; then
+      echo matched;
+      exists=false
+      break
+    fi
+  done
+  if [ "$exists" = true ]; then
+    sitesscriptstoremove+=("$script")
+  fi
+
+done < <(o365 spo sitescript list -o json | jq -c '.[]')
+
+printf '%s\n' "${sitesscriptstoremove[@]}"
+echo "Press Enter to start deleting (CTRL + C to exit)"
+read foo
+
+for script in "${sitesscriptstoremove[@]}"; do
+    scriptTitle=$(echo ${script} | jq -r '.Title')
+    scriptId=$(echo ${script} | jq -r '.Id')
+    echo "Deleting Site design..."  $scriptTitle $scriptId
```

Comment:
This produces string like `Deleting Site design... Contoso1 ba43a4af-c87e-4cc2-a3a6-91cca1519f84`. It would be more readable if the string was `Deleting site design Contoso1...`